client/doc.go

```diff
@@ -0,0 +1,156 @@
+/*
+Package client provides a library for all client-side CONIKS operations.
+
+Introduction
+
+One crucial component of the CONIKS key management system are the clients
+verifying the cryptographic proofs returned by the CONIKS server as part of
+the registration, lookup and monitoring protocols. client implements the
+operations performed by CONIKS clients during these protocols. This document
+outlines each of these protocols from a client perspective.
+
+For the server-side specification of the CONIKS protocols, see
+https://godoc.org/github.com/coniks-sys/coniks-go/server
+
+Registration
+
+Prerequisites: This protocol requires that the client pin both the
+server's public signing key for verifying signed tree roots (STRs) and
+the server's initial private index verification key (i.e. the public
+VRF key). We assume a separate PKI exists to manage the server's keys.
```

Comment:
> No, this sentence isn't about TLS

So then what other PKI is necessary besides the pinning?  I guess this is the unresolved questions in #47 and #52.

> Or maybe I'm misremembering what we had decided about which VRF key is included in the policy?

I think you're confusing the two keys.  It's the next signing key that we're considering also including in the policy.  The VRF corresponds to a particular epoch so doesn't require any more management.